guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,275 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
 # Funding a wallet
 
-Common problems faced by new users include buying their first bitcoin and transferring it to a new wallet. For this, most will purchase their first bitcoin from an exchange using a national or fiat currency. This generally involves providing some form of documentation (such as ID) and already having a bank account. After purchasing their bitcoin, they can then send it to an external wallet. While this works, there are better alternatives available. Understanding how your users fund their wallets can help you create more tailored and dedicated UI flows. Certain ways users fund their wallets are outlined below.
+A user will need to fund their wallet with some bitcoin to begin using it. When the user funds a wallet for the first time they are likely to be introduced to new concepts that need to be clearly explained. The design flows around funding a wallet can range from simple to complex, depending on your application's target audience and which services you offer to streamline the process.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Ask the user if they own bitcoin."
+   alt-text = "Screen asking the user if they own any bitcoin."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
```

Comment:
This screen is very busy and lacks a clear call-to-action for newcomers. I'd put up a big "Send your bitcoin to the address below" title. I'd also remove the completeness indicator, makes me think that I can't send the bitcoin yet. Also think the note about the channel and the "What is a channel?" link should be right next to each other.

Reply:
Made some changes, what do you think? 

I think letting the user know that their funds won't appear until that sync is complete is very important. I re-worded it so it's clear you can still fund while it is in progress. 

Having the user think their funds have gone missing is a bigger problem to avoid than thinking they can not fund straight away. Though as mentioned above you can explain that they can still fund while the sync is in progress with some copy.

Reply:
Quick mock-up for simplifying this.

<img width="1050" alt="Frame 2949" src="https://user-images.githubusercontent.com/695901/147095287-3e94b3a5-9b84-4b1f-b3e5-63c6dd623ab0.png">

Reply:
@GBKS @Bosch-0 Should we update the mock or is this something we can want to do in a follow-up? Would be cool if we can get this baby merged if everything else looks good. I like the simplified version.

Reply:
I think this misses quite a lot of important things - if a channel is being automatically opened the user will have less funds available once that happens. If it is not explained that a channel will be opened I'd imagine they would be confused as to why they received less. I like the way the network sync is presented though. I'd have to change quite a few things if I used the above image so maybe add these in a follow up? idm 

Can you share the Figma file for this @GBKS

Reply:
Yup, just asked @GBKS and we agreed this is better suited for a follow-up and some explorations. We can even invite people to post their own solutions in a workshop kind of. format after providing a case for it. Just an idea, but can be just a simple follow up PR. 

Let's merge this baby 🚀

Reply:
I think that channel opening info + fee should be explained in a separate page that comes before this invoice screen. It's just too much for a single screen.